proposals/vip_12.ts

```diff
@@ -0,0 +1,17 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const vip_x: ProposalDescription = {
+  title: 'VIP-12: Add new PSMs to PCV Guardian Whitelist',
+  commands: [
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'addWhitelistAddresses(address[])',
+      arguments: [['{voltV2DaiPriceBoundPSM}', '{voltV2UsdcPriceBoundPSM}']],
+      description: 'Add new DAI, and USDC PSMs to PCV Guardian whitelist'
+    }
+  ],
```

Comment:
Pre-Proposal action: pull all VOLT out of all PSM's

Record how much outstanding VOLT exists

Proposal should mint the proper amount of new Volt to transfer to the VoltMigrator contract based on outstanding VOLT at time of proposal